download/releaseNotes/releaseNotes7.2.adoc

```diff
@@ -0,0 +1,69 @@
+= Release Notes 7.2
+:awestruct-description: New and noteworthy, demos and status for OptaPlanner 7.2.
+:awestruct-layout: releaseNotesBase
+:awestruct-priority: 1.0
+:awestruct-release_notes_version: 7.2
```

Comment:
IIRC, the generated release notes page will have a link to upgrade recipe 7.2 page, which will give a 404 if it doesn't exist yet. Please create an empty one? Probably easiest to copy the 6.5 upgrade recipe - that's empty too and has some wording around it.